tests/compute/subresources/expandspec.go

```diff
@@ -174,7 +173,7 @@ var _ = compute.SIGDescribe("ExpandSpec subresource", func() {
 
 				_, err := virtClient.ExpandSpec(testsuite.GetTestNamespace(vm)).ForVirtualMachine(vm)
 				Expect(err).To(HaveOccurred())
-				Expect(err).To(MatchError(fmt.Sprintf(instancetypepkg.VMFieldsConflictsErrorFmt, instancetypepkg.Conflicts{field.NewPath("spec.template.spec.domain.resources.requests.memory")})))
+				Expect(err).To(MatchError(conflict.New("spec.template.spec.domain.resources.requests.memory").Error()))
```

Comment:
nit: You can drop `Expect(err).To(HaveOccurred())`, `MatchError` covers that already.